### PR TITLE
Add missing map commands [CLC-200]

### PR DESCRIPTION
**Pull request by mtyazici**
Added following commands:
- map lock
- map try-lock
- map unlock
- map values

### CODE REVIEW

File: base/commands/map/map_lock.go [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+	lidv := ctx.Value(lockIDKey{})
+	if lidv == nil {
+		return defaultLockID
+	}
+	lid, ok := lidv.(lockID)
+	if !ok {
+		return defaultLockID
+	}
+	return int64(lid)
+}
```
Comment (kutluhanmetin) [inline base/commands/map/map_lock.go:87]:
```suggestion
```

**Review by kutluhanmetin — DISMISSED** [review]
Just a minor suggestion

**Review by kutluhanmetin — DISMISSED** [review]

File: docs/modules/ROOT/pages/clc-map.adoc [hunk 2]
```diff
@@ -239,6 +243,184 @@ Example:
...
+|`--ttl`
+|Optional
+|Duration in milliseconds after which the lock will expire.
+|-1
+
+Example:
+
+[source,bash]
+----
+clc map lock -n myMap --key-type string key1
```
Comment (yuce) [inline docs/modules/ROOT/pages/clc-map.adoc:331]:
```suggestion
clc map lock -n my-map --key-type string key1
```

File: docs/modules/ROOT/pages/clc-map.adoc [hunk 3]
```diff
@@ -239,6 +243,184 @@ Example:
...
+|Duration in milliseconds after which the lock will expire.
+|-1
+
+|===
+
+Example:
+
+[source,bash]
+----
+clc map try-lock -n myMap --key-type string key1
```
Comment (yuce) [inline docs/modules/ROOT/pages/clc-map.adoc:379]:
```suggestion
clc map try-lock -n my-map --key-type string key1
```

**Review by yuce — CHANGES_REQUESTED** [review]
Just trivial ideas...

**Review by yuce — APPROVED** [review]
Looks good!

**Pull request merged**